ansible/site.yml

```diff
@@ -30,4 +30,6 @@
     - rpi-update
     - screenly
     - splashscreen
+    - enable-nginx
```

Comment:
Feels a bit award to have both. Let's merge them into one role called `nginx` and mark it up properly with comments.